app/src/main/java/com/kpiroom/bubble/util/databinding/ProgressState.kt

```diff
@@ -0,0 +1,7 @@
+package com.kpiroom.bubble.util.databinding
+
+enum class ProgressState {
```

Comment:
remove this. Place this constants in companion object in ProgressStateContainer instead:
```
companion object {
const val LOADING = 3432
const val ....
}
```